frontend/package.json

```diff
@@ -6,20 +6,19 @@
     "node": "18"
   },
   "dependencies": {
-    "@apollo/client": "3.4.17",
+    "@apollo/client": "^3.7.2",
     "@fortawesome/fontawesome-svg-core": "^6.2.0",
     "@fortawesome/free-solid-svg-icons": "^6.2.0",
     "@fortawesome/react-fontawesome": "^0.2.0",
     "@microsoft/applicationinsights-react-js": "^3.0.5",
     "@microsoft/applicationinsights-web": "^2.5.10",
     "@szhsin/react-menu": "^1.5.0",
-    "@testing-library/jest-dom": "^5.16.1",
-    "@testing-library/react": "^12.1.2",
-    "@testing-library/user-event": "^13.5.0",
-    "@trussworks/react-uswds": "^3.2.0",
+    "@testing-library/react": "^13.4.0",
+    "@testing-library/user-event": "14.1.1",
+    "@trussworks/react-uswds": "3.2.0",
     "@types/google-libphonenumber": "^7.4.23",
     "@types/history": "^4.7.10",
-    "@types/react-router-dom": "^5.3.3",
+    "@types/react-router-dom": "5.3.3",
     "@types/react-transition-group": "^4.4.5",
     "@types/testing-library__jest-dom": "^5.14.2",
```

Comment:
Can we set `@types/testing-library__jest-dom` to `"^5.14.5"` to match the update we are doing to `@testing-library/jest-dom` on line 199. 🤔 

haha only noticed it due to this dependabot PR: https://github.com/CDCgov/prime-simplereport/pull/4918